tests/feature/test_scenarios.py

```diff
@@ -0,0 +1,59 @@
+"""Test scenarios shortcut."""
+import textwrap
+
+
+def test_scenarios(testdir):
+    """Test scenarios shortcut."""
+    testdir.makeconftest("""
+        import pytest
+        from pytest_bdd import given
+
+        @given('I have a bar')
+        def i_have_bar():
+            print('bar!')
```

Comment:
This seems to be done via the outer test (looking at the output).
Or is there something specific to assert in the inner test?